app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -344,6 +345,10 @@ function AcuantCapture(
   // There's some pretty significant changes to this component when it's used for
   // selfie capture vs document image capture. This controls those changes.
   const selfieCapture = name === 'selfie';
+  // When it's the back of the ID we want to log information about the camera
+  // This hook does that.
+  const isBackOfId = name === 'back';
```

Comment:
Wondering if device OS is in another language, will it be `back`.

Mentioning this because https://github.com/Acuant/JavascriptWebSDKV11/blob/6a4f2b8648df20959a57c89e8f26dd3630bd86bb/webSdk/AcuantCamera.js#L629-L670